adapter/ipListChecker/cfg.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 Google Inc.
```

Comment:
That is how we had it in the prototype. 
1. Since we (+ adapter authors) are fully bought into defining everything thru protobufs, it forces uniformity.
2. It restricts the types of data structures that can be used for config. proto structs are a subset of what can be expressed in go.

I agree that it is a pain though; if we cannot come up with a very easy proto compile story; then we will revert to go reflection only.